DFC.ServiceTaxonomy.Taxonomies/Views/Admin/Edit.cshtml

```diff
@@ -19,3 +19,36 @@
     <input type="hidden" name="TaxonomyContentItemId" value="@Model.TaxonomyContentItemId" />
     <input type="hidden" name="TaxonomyItemId" value="@Model.TaxonomyItemId" />
 </form>
+
+@if (contentTypeDefinition?.DisplayName == "Page Location")
+{
+    <script depends-on="jQuery" at="Foot">
```

Comment:
Don't think it'd be too much effort to move this into the PageLocation module and add the script from the PageLocation part's view. We should keep the taxonomies module generic.